web/themes/joinup/templates/navigation/menu-local-tasks.html.twig

```diff
@@ -12,10 +12,16 @@
  */
 #}
 {% if primary %}
+  <button id="local-tasks-primary" class="mdl-button mdl-js-button mdl-button--icon">
```

Comment:
Please check if the check can be simpler such as:

```
  {% set type = (primary ? 'primary' : 'secondary') %}
  <button id="local-tasks-{{ type }}" class="mdl-button mdl-js-button mdl-button--icon">
    <span class="icon icon--dots"></span>
  </button>
  <h2 class="visually-hidden">{{ 'Tabs navigation'|t }}</h2>
  <ul class="mdl-menu mdl-menu--bottom-right mdl-js-menu mdl-js-ripple-effect" for="local-tasks-{{ type }}">{{ primary ?: secondary }}</ul>
```

The difference seems small enough to allow a shorthand.